src/dbwrapper.cpp

```diff
@@ -42,7 +42,7 @@ class CBitcoinLevelDBLogger : public leveldb::Logger {
                     va_list backup_ap;
                     va_copy(backup_ap, ap);
                     // Do not use vsnprintf elsewhere in bitcoin source code, see above.
-                    p += vsnprintf(p, limit - p, format, backup_ap);
+                    p += std::vsnprintf(p, limit - p, format, backup_ap);
```

Comment:
This change is with intent or by accident? PR title says "use std:: for mem*( funcs", this one isn't mem* func. Either title should be changed or this removed from this PR. Also, if this is changed, `#include <stdint.h>` at the beginning of file should be changed to `#include <cstdio>`.

Reply:
> #include <stdint.h> at the beginning of file should be changed to #include <cstdio>

`stdint.h` is still required, and `<cstdio>` can just be added.

I don't think this change needs to be dropped, or it's inclusion is confusing. I've added a note to the PR body.

Reply:
Yes, my mistake, of course `<cstdio>` does no replace `<stdint.h>`.